packages/common/global-types/kaltura-player/player-config.d.ts

```diff
@@ -10,195 +10,205 @@
+      audioLanguage?: string;
+      textLanguage?: string;
+      useNativeTextTrack?: boolean;
+      enableCEA708Captions?: boolean;
+      captionsTextTrack1Label?: string;
+      captionsTextTrack1LanguageCode?: string;
+      captionsTextTrack2Label?: string;
+      captionsTextTrack2LanguageCode?: string;
+      volume?: number;
+      startTime?: number;
+      playsinline?: boolean;
+      preload?: string;
+      autoplay?: boolean;
+      loop?: boolean;
+      allowMutedAutoPlay?: boolean;
+      muted?: boolean;
+      pictureInPicture?: boolean;
+      options?: Options;
+      preferNative?: PreferNative;
+      inBrowserFullscreen?: boolean;
+      playAdsWithMSE?: boolean;
+      streamPriority?: StreamPriority[];
+    }
+
+    export interface Options {
+      html5?: Html5;
+    }
+
+    export interface Html5 {
+      hls?: Network;
+      dash?: Network;
+      native?: Network;
+    }
+
+    export interface PreferNative {
+      hls?: boolean;
+      dash?: boolean;
+    }
+
+    export interface StreamPriority {
+      engine?: string;
+      format?: string;
+    }
+
+    export interface Provider {
+      env: Env;
+      partnerId?: string;
+      uiConfId?: string;
+      ks: string;
+      log?: Log;
+    }
+
+    export interface Env {
+      serviceUrl: string;
+      cdnUrl?: string;
+      statsServiceUrl?: string;
+      liveStatsServiceUrl?: string;
+      analyticsServiceUrl?: string;
+      apiFeatures?: APIFeatures;
+    }
+
+    export interface APIFeatures {
+      entryRedirect?: string;
+    }
+
+    export interface Session {
+      userId?: string;
+      isAnonymous?: boolean;
+      partnerId?: string;
+      uiConfId?: string;
+      ks?: string;
+      id?: string;
+    }
+
+    export interface Sources {
+      options?: Network;
+      hls?: Dash[];
+      dash?: Dash[];
+      progressive?: Progressive[];
+      id: string;
+      duration?: number;
+      type: EntryTypes;
+      poster?: string;
+      dvr?: boolean;
+      vr?: null;
+      metadata?: Metadata;
+      captions?: any[];
+    }
+
+    export interface Dash {
+      id?: string;
+      url?: string;
+      mimetype?: string;
+    }
+
+    export interface Metadata {
+      name?: string;
+      description?: string;
+      tags?: string;
+    }
+
+    export interface Progressive {
+      id?: string;
+      url?: string;
+      mimetype?: string;
+      bandwidth?: number;
+      width?: number;
+      height?: number;
+      label?: string;
+    }
+
+    export interface UI {
+      stargetId?: string;
+      debug?: boolean;
+      log?: Log;
+    }
+
+    export interface Contrib {
+      ui: {
+        kitchenSink: KitchenSinkConfig;
+        fonts: ContribFonts;
+      };
+    }
+
+    export interface ContribFonts {
+      fontFamily: string;
+      testingFont?: TestingFontOptions;
+      downloadData?: {
+        name: string;
+        url: string;
+      };
+    }
+
+    export interface KitchenSinkConfig {
```

Comment:
1. in `contrib-plugin-configs.ts` adjust `get playerConfig(): KalturaPlayerTypes.PlayerConfig | ContribConfig`
2. in the same file add
```

interface ContribConfig {
  contrib?: {
    ui?: {
      ....
    }
  } 
} 
```
and move relevant parts that both you and Shai wrote into this interface

@ainvoner fyi

Reply:
done